File: .claude/skills/agent-skill-kit-commons/packages/mjs/src/optimize/providers/none.mjs
import { generateDeterministic } from "../generate.mjs";
import { proposeVariantsDeterministic } from "../rewrite.mjs";

export const PROVIDER_NAME = "none";

export async function generate(parsed, opts) {
  return generateDeterministic(parsed, opts.queryCount ?? 8);
}

export async function propose(parsed) {
  return proposeVariantsDeterministic(parsed);
}

File: .claude/skills/agent-skill-kit-commons/packages/py/src/skills_ref/optimize/providers/none.py
from __future__ import annotations

from ..generate import generate_deterministic
from ..rewrite import propose_variants_deterministic

PROVIDER_NAME = "none"


def generate(parsed, opts):
    return generate_deterministic(parsed, opts.get("query_count", 8))


def propose(parsed, opts=None):
    return propose_variants_deterministic(parsed)
